submissions/Lu-sure/post-oop/index.js

```diff
@@ -0,0 +1,162 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+  // ======== OBJECTS DEFINITIONS ========
+  class Inhabitant {
+    constructor(name, age, spesies) {
+      this.name = name;
+      this.age = age;
+      this.species = spesies;
+    }
+
+    getName() {
+      return this.name;
+    }
+
+    introduce() {
+      return `I'm ${this.age} years old ${this.species}.`
+    }
+  }
+
+  class Human extends Inhabitant {
+    constructor(name, lastName, age, gender, saying) {
+      super(name, age, 'human');
+      this.gender = gender;
+      this.lastName = lastName;
+      this.saying = saying;
+    }
+
+    getName() {
+      return `${this.name} ${this.lastName}`;
```

Comment:
Why not to follow `introduce` re-use/overloading pattern?

Reply:
I don't know the reason, so I made changes))
Check if correct, please.
I need a space between first and last name, didn't found better way than using template literals. If there are a better way to add that space, please, tell me